src/indent.c

```diff
@@ -941,9 +948,23 @@ get_breakindent_win(
     // Add offset for number column, if 'n' is in 'cpoptions'
     bri += win_col_off2(wp);
 
+    // add additional indent for numbered lists
+    if (wp->w_briopt_list > 0)
+    {
+	regmatch_T	    regmatch;
+	regmatch.regprog = vim_regcomp(curbuf->b_p_flp,
+		RE_MAGIC + RE_STRING + RE_AUTO + RE_STRICT);
+	if (regmatch.regprog != NULL && vim_regexec(&regmatch, line, 0))
+	{
+	    vim_regfree(regmatch.regprog);
+	    bri += wp->w_briopt_list;
```

Comment:
Should it be smth like
```
›       ›       bri += wp->w_briopt_list > 0 ? wp->w_briopt_list : len(regmatch???);
```
provided there is another condition in the outer `if (wp->w_briopt_list > 0 || wp->w_briopt_list == -1)`

For the `formatlistpat` indentation?

Reply:
![image](https://user-images.githubusercontent.com/234774/126277547-0f28555a-897d-4e87-be5b-6595f5889ee4.png)

having formatlistpat and `set briopt+=list:-1` + 
```
    if (wp->w_briopt_list > 0 || wp->w_briopt_list == -1)
    {
›       regmatch_T›         regmatch;

›       regmatch.regprog = vim_regcomp(curbuf->b_p_flp,
›       ›       ›       ›          RE_MAGIC + RE_STRING + RE_AUTO + RE_STRICT);
›       if (regmatch.regprog != NULL)
›       {
›           if (vim_regexec(&regmatch, line, 0))
›           ›   if (wp->w_briopt_list > 0)
›       ›           bri += wp->w_briopt_list;
›       ›       else
›       ›           bri += (*regmatch.endp - *regmatch.startp);
›           vim_regfree(regmatch.regprog);
›       }
    }

```